lib/simp/beaker_helpers.rb

```diff
@@ -165,7 +165,7 @@ def copy_fixture_modules_to( suts = hosts, opts = {})
     opts[:pluginsync] = opts.fetch(:pluginsync, true)
 
     unless ENV['BEAKER_copy_fixtures'] == 'no'
-      Array(suts).each do |sut|
+      block_on(suts, run_in_parallel: true) do |sut|
```

Comment:
The predominant style in this file is :symbol => value, not symbol: value.  Please follow the predominant style.